### ISSUE TITLE
AE2’s ME formation planes (normal and fluid) don’t work most of the time

### ISSUE THREAD

**Issue by Dimentive** [NillerMedDild/Enigmatica2#426]
#### Versions
* **Enigmatica 2:** [1.63a](https://minecraft.curseforge.com/projects/enigmatica2/files/2691579) (latest)
* **Forge:** 14.23.5.2781
* **Applied Energistics 2:** [rv6-stable-6](https://minecraft.curseforge.com/projects/applied-energistics-2/files/2652453) (latest)

**Can the issue be reproduced?**
Yes

#### Issue

Like you know from Discord, Niller, ME formation planes don’t always work in Enigmatica 2. In my survival world, the two formation planes (block and fluid) didn’t work at all. In my creative test world, they didn’t work either until I made a _second_ test ME setup, where they _did_ work, yet they still didn’t work in my first setup. I then deleted my first setup, and suddenly, the _block_ formation plane started working but not the fluid formation plane. This is one of the strangest bugs I’ve ever encountered, and I don’t have a clue what could be causing it.

Apparently, it’s _not_ a problem with [Applied Energistics 2](https://minecraft.curseforge.com/projects/applied-energistics-2), as two people [on the AE2 issue tracker](https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/3993) told me that it worked just fine for them. On the Enigmatica 2 Discord server, someone else playing the same modpack tested it and couldn’t get it to work, which suggest that something in Enigmatica 2 is causing formation planes to not work (unless you make a second ME network, which makes very little sense).

There might be a conflict between one of the mods—maybe a mod that modifies the spawning of things or one of the Applied Energistics 2 add-ons ([AE2 Stuff](https://minecraft.curseforge.com/projects/ae2-stuff) or [ExtraCells2](https://minecraft.curseforge.com/projects/extracells2)). I have experienced a few bugs with AE2 Stuff, and it hasn’t been updated in one and a half years, so it could be that mod, although the formation planes are from AE2, not AE2 Stuff, and AE2 Stuff shouldn’t affect AE2’s blocks but rather adds its own blocks. I’m honestly not sure what could possibly be causing it, but nevertheless, it’s seemingly a problem with one of the mods in Enigmatica 2, rather than Applied Energistics 2 on its own.

**In my survival world:**
![Formation plane](https://user-images.githubusercontent.com/34230933/55903757-aae47400-5bce-11e9-9374-a92fc608636f.png)

**First ME setup in my creative test world – neither formation plane is working:**
![wot](https://user-images.githubusercontent.com/34230933/55903756-aae47400-5bce-11e9-9288-f634956a7225.png)

**Second ME setup in my creative test world – both formation planes are working:**
![And NOW it works, apparently… but only here](https://user-images.githubusercontent.com/34230933/55903755-aa4bdd80-5bce-11e9-8e62-d6e17efea505.png)

**Comment by NillerMedDild** [comment]
I'm guessing this is some ExtraCells2 weirdness, but it's hard to say.

**Comment by Natchel93** [comment]
O I got this working! I use it for Auto Plastic production.
You have to set up the formation plane to Highest Priority with it filtered to that item while making sure that That item has somewhere to Physically pull from it in the AE2 System. ![ME Interface](https://user-images.githubusercontent.com/33611567/56446690-f5de4580-62b8-11e9-9d88-740cb38eaf48.png) For instance I have mine set up working though a ME Interface which has wood asked from the system held in it.
![Formation plane](https://user-images.githubusercontent.com/33611567/56446753-4bb2ed80-62b9-11e9-8eee-780d66c02bfe.png)
![Priority](https://user-images.githubusercontent.com/33611567/56446756-4d7cb100-62b9-11e9-8e28-d679a3dfd3fb.png)

Thats the only way i was able to get it working 

https://www.youtube.com/watch?v=gBeC6-Braws
This Youtube Video helped with that

**Comment by Dimentive** [comment]
@Natchel93 I didn’t do anything wrong. I already set them to the highest priority and had items both in my system and currently going in my system.

It’s a strange bug that is likely caused by ExtraCells2, but I’m not sure.
As mentioned in the issue, I did get the block and fluid formation planes working in the second setup but not the first one, and I got only the block formation plane working in the third setup but not the fluid formation plane. It’s very odd.
